src/nodes.jl

```diff
@@ -266,6 +266,9 @@ Base.next(it::XMLElementIter, p::Xptr) = (XMLElement(p), ccall((:xmlNextElementS
 
 child_elements(x::XMLElement) = XMLElementIter(x.node.ptr)
 
+has_child_elements(x::XMLElement) = !isempty(child_elements(x))
```

Comment:
Given how simple and readable is `!isempty(child_elements(x))`, I'm not sure a function is needed here, especially since it's used in a single place.